mailet/base/src/test/java/org/apache/mailet/base/test/MimeMessageBuilder.java

```diff
@@ -187,18 +186,6 @@ public BodyPart build() throws IOException, MessagingException {
         }
     }
 
-    public static MimeMessage defaultMimeMessage() {
```

Comment:
these methods actually makes sense in a builder as they are factory methods

Reply:
No, I do not agree, they do not depend on _MimeMessageBuilder_, they can be thought as helpers.